src/services/config.js

```diff
@@ -156,6 +159,26 @@ class ConfigData {
     // Trim whitespace if given a non-null string.
     return string ? string.trim() : string;
   }
+
+  addApiKeysToProfile(profile) {
+    if (profile) {
+      const credentials = this.getApiKeysByProfileID(profile.id);
+      if (credentials) {
+        profile.apiKey = credentials.apiKey;
+        profile.apiSecret = credentials.apiSecret;
+      }
+    }
+  }
+
+  getApiKeysByProfileID(profileId) {
+    if (this.profiles) {
```

Comment:
This can be removed since it should always be present.

Reply:
Thanks for pointing. Updated the change.